aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1994,9 +1994,21 @@ Tensor& orgqr_kernel_impl(Tensor& result, const Tensor& tau) {
     TORCH_CHECK(false, "Calling torch.orgqr on a CUDA tensor requires compiling ",
       "PyTorch with cuSOLVER. Please use PyTorch built with cuSOLVER support.");
   #endif
-  }
+}
+
+REGISTER_DISPATCH(orgqr_stub, &orgqr_kernel_impl);
+
+void ormqr_kernel(const Tensor& input, const Tensor& tau, const Tensor& other, bool left, bool transpose) {
+#if defined(USE_CUSOLVER)
+  return ormqr_cusolver(input, tau, other, left, transpose);
```

Comment:
Is this return within a void function correct?

Reply:
It is a valid code, but you're right, it shouldn't be here.